pkgs/development/tools/click/default.nix

```
@@ -0,0 +1,92 @@
+{ lib
+, fetchFromGitLab
+, buildPythonPackage
+, autoconf
```

Can be called automatically.

```suggestion
, autoreconfHook
```